internal/dialect/dialectquery/clickhouse.go

```diff
@@ -42,3 +42,46 @@ func (c *Clickhouse) GetLatestVersion(tableName string) string {
 	q := `SELECT max(version_id) FROM %s`
 	return fmt.Sprintf(q, tableName)
 }
+
+type ClickhouseReplicated struct {
+	ClusterName string
+}
+
+var _ Querier = (*ClickhouseReplicated)(nil)
+
+func (c *ClickhouseReplicated) CreateTable(tableName string) string {
+	q := `CREATE TABLE IF NOT EXISTS %s ON CLUSTER '%s' (
+		version_id Int64,
+		is_applied UInt8,
+		date Date default now(),
+		tstamp DateTime default now()
+	  )
+	  ENGINE = ReplicatedMergeTree()
```

Comment:
Would be awesome to be able to pass keeper path here as well, like

`ENGINE = ReplicatedMergeTree('/clickhouse/tables/common/migrations', '{replica}')
`

Reply:
There's an endless number of ways to configure clickhouse, which means goose will need to know about all these and expose knobs for them.

I think the most practical solution would be for projects to implement the `Store` interface, e.g.,

https://github.com/pressly/goose/pull/520#issuecomment-2419464112